test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsTestFixture.cs

```diff
@@ -33,7 +29,7 @@ public ApplicationInsightsTestFixture(string scriptRoot, string testId)
                 SecretsPath = Environment.CurrentDirectory, // not used
                 IsAuthDisabled = true
             };
-          //  WebApiConfig.Register(_config, _settingsManager, HostSettings);
+            //  WebApiConfig.Register(_config, _settingsManager, HostSettings);
```

Comment:
Are these tests disabled or something?

Reply:
Yeah, they're disabled. There was a bunch of 'unsupported-in-.NET-core' stuff in App Insights so the tests didn't work correctly. They've just released a beta with all of this ported, so I have it on my plate to re-enable all of our features and tests.